packages/manager/apps/pci-object-storage/src/pages/objects/container/object/replication/ReplicationList.page.tsx

```diff
@@ -0,0 +1,281 @@
+  Notifications,
+  PciGuidesHeader,
+  PciMaintenanceBanner,
+  useNotifications,
+  useProductMaintenance,
+  useProjectUrl,
+} from '@ovh-ux/manager-react-components';
+import { useTranslation } from 'react-i18next';
+import { useBytes, useProject } from '@ovh-ux/manager-pci-common';
+
+import {
+  OdsBreadcrumb,
+  OdsButton,
+  OdsBreadcrumbItem,
+  OdsSpinner,
+  OdsText,
+  OdsMessage,
+} from '@ovhcloud/ods-components/react';
+
+import { useServerContainer } from '@/api/hooks/useContainer';
+import { useDatagridColumn } from './useDatagridColumn';
+
+import { useGetRegion } from '@/api/hooks/useRegion';
+import {
+  useAllServerStorages,
+  useStorage,
+  useStorageEndpoint,
+} from '@/api/hooks/useStorages';
+
+import '../show/style.scss';
+
+import { TContainer } from '../show/Show.page';
+import { STATUS_ENABLED } from '@/constants';
+import { useStorageFeatures } from '@/hooks/useStorageFeatures';
+
+export default function ReplicationListPage() {
+  const { storageId, projectId } = useParams();
+
+  const [searchParams] = useSearchParams();
+  const { data: project } = useProject();
+
+  const { hasMaintenance, maintenanceURL } = useProductMaintenance(
+    project?.project_id,
+  );
+
+  const { formatBytes } = useBytes();
+
+  const hrefProject = useProjectUrl('public-cloud');
+
+  const { t } = useTranslation(['objects', 'containers/replication']);
+  const navigate = useNavigate();
+
+  const objectStorageHref = useHref('..');
+  const containerDetailsHref = useHref(
+    `../${storageId}?region=${searchParams.get('region')}`,
+  );
+
+  const enableVersioningHref = useHref(
+    `./enableVersioning?region=${searchParams.get('region')}`,
+  );
+
+  const { data: region } = useGetRegion(
+    project?.project_id,
+    searchParams.get('region'),
+  );
+
+  const { storage: targetContainer } = useStorage(
+    project?.project_id,
+    storageId,
+    searchParams.get('region'),
+  );
+
+  const { url } = useStorageEndpoint(project?.project_id, targetContainer);
+
+  const { data: serverContainer, isLoading } = useServerContainer(
+    project?.project_id,
+    searchParams.get('region'),
+    targetContainer?.name,
+    targetContainer?.id,
+  );
+
+  const container = useMemo((): TContainer => {
+    if (!serverContainer) return undefined;
+    const s3StorageType = targetContainer?.s3StorageType;
+
+    return {
+      ...serverContainer,
+      id: serverContainer?.id || targetContainer?.id,
+      name: serverContainer?.name || targetContainer?.name,
+      objectsCount:
+        serverContainer?.storedObjects || serverContainer?.objectsCount,
+      usedSpace: formatBytes(
+        serverContainer?.storedBytes || serverContainer?.objectsSize,
+        2,
+        1024,
+      ),
+      publicUrl: url,
+      s3StorageType,
+      regionDetails: s3StorageType ? region : undefined,
+      staticUrl: serverContainer?.staticUrl || serverContainer?.virtualHost,
+    };
+  }, [serverContainer, region, targetContainer, url]);
```

Comment:
This is exactly the same code than in `DeleteReplication.page.tsx`, please consider refactoring